### PR TITLE
Fix remaining webgl + CAN_ADDRESS_2GB issues

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: src/library_webgl2.js [hunk 1]
```diff
@@ -622,23 +622,23 @@ var LibraryWebGL2 = {
     assert((value & 3) == 0, 'Pointer to integer data passed to glClearBufferiv must be aligned to four bytes!');
 #endif
 
-    GLctx.clearBufferiv(buffer, drawbuffer, HEAP32, value>>2);
+    GLctx.clearBufferiv(buffer, drawbuffer, HEAP32, {{{ getHeapOffset('value', 'float') }}});
```
Comment (kripken) [inline src/library_webgl2.js:625]: This and some others later are not floats (but they are 4 bytes, so it happens to work).

File: src/library_webgl.js [hunk 2]
```diff
@@ -4017,10 +4017,10 @@ for (/**@suppress{duplicate}*/var i = 0; i < {{{ GL_POOL_TEMP_BUFFERS_SIZE }}};
     GLctx.multiDrawWebgl['multiDrawElementsWEBGL'](
       mode,
       HEAP32,
-      counts >> 2,
+      {{{ getHeapOffset('counts', 'float') }}},
       type,
       HEAP32,
-      offsets >> 2,
+      {{{ getHeapOffset('offsets', 'float') }}},
```
Comment (kripken) [inline src/library_webgl.js:4023]: These two look wrong.

If you can please fix these with a new commit (so I don't need to read the entire file again) that would be appreciated!

File: test/test_browser.py [hunk 3]
```diff
@@ -4622,6 +4625,8 @@ def test_webgl_from_client_side_memory_without_default_enabled_extensions(self):
   # For testing WebGL draft extensions like this, if using chrome as the browser,
   # We might want to append the --enable-webgl-draft-extensions to the EMTEST_BROWSER env arg.
   @requires_graphics_hardware
+  @no_2gb('multiDrawElementsInstancedWEBGL: size exceeds the supported range')
+  @no_4gb('multiDrawElementsInstancedWEBGL: size exceeds the supported range')
```
Comment (kripken) [inline test/test_browser.py:4629]: just out of curiosity, is this a browser limitation?

**Review by kripken — COMMENTED** [review]
lgtm % comment

File: test/test_browser.py (hunk 3 above)
Comment (sbc100) [reply test/test_browser.py:4629]: It seems to be yes. @kainino0x I this a known limitation?   Should I open a chromium bug?

**Review by kripken — APPROVED** [review]

File: test/test_browser.py (hunk 3 above)
Comment (kainino0x) [reply test/test_browser.py:4629]: I think we just missed this when we updated all the other WebGL entry points. Filed https://crbug.com/324562920

**Pull request merged**